Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -216,24 +216,29 @@ private let learnMoreTextViewStyle: TextViewStyle = { (textView: UITextView) ->
   return textView
 }
 
-private func attributedLearnMoreText() -> NSAttributedString {
-  let string = """
-  \(Strings.Kickstarter_is_not_a_store_Its_a_way_to_bring_creative_projects_to_life())
-  \(Strings.Learn_more_about_accountability())
-  """ as NSString
-
-  let linkRange = string.range(of: Strings.Learn_more_about_accountability())
-  let stringRange = string.range(of: string as String)
-
-  let attributedString = NSMutableAttributedString(string: string as String)
-
-  let url = urlForHelpType(
-    HelpType.trust, baseUrl: AppEnvironment.current.apiService.serverConfig.webBaseUrl
+private func attributedLearnMoreText() -> NSAttributedString? {
+  // swiftlint:disable line_length
+  let string = localizedString(
+    key: "Kickstarter_is_not_a_store_Its_a_way_to_bring_creative_projects_to_life_Learn_more_about_accountability",
+    defaultValue: "<p>Kickstarter is not a store. It's a way to bring creative projects to life.</br><a href=\"https://www.kickstarter.com/trust\">Learn more about accountability</a><p>"
   )
+  // swiftlint:enable line_length
+
+  guard let attributedString = try? NSMutableAttributedString(
+    data: Data(bytes: string.utf8),
+    options: [.documentType: NSAttributedString.DocumentType.html],
+    documentAttributes: nil
+  ) else { return nil }
+
+  let attributes: [NSAttributedString.Key: Any] = [
+    .font: UIFont.ksr_caption1(),
+    .foregroundColor: UIColor.ksr_text_dark_grey_500,
+    .underlineStyle: 0
```

Comment:
`NSUnderlineStyle.none` is buggy in Swift 4.2 🤷‍♂ 
https://bugs.swift.org/browse/SR-8184?page=com.atlassian.jira.plugin.system.issuetabpanels%3Achangehistory-tabpanel

Reply:
Also, I had to apply this here to the whole string, it was having no affect in the `UITextView`'s `linkTextAttributes` property 🤦‍♂ . I think this means that if we were wanting to be any more advanced with our links, like say let them have a different font or something, we may have to take a more manual approach.